### PR TITLE
Start back office backend and basic stats

### PR DESCRIPTION
**Pull request by zalexki**
@NeOMakinG 
this is a first draft of back end configuration and stats that you can use to start the front end.

ping me if needed i explained pablo everything i did, new feature like cache refresh for stats is still missing but i thought you could at least start the integration (on this branch or we can merge this and start from new-module).

### CODE REVIEW

File: src/ObjectModel/Statistics.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace PrestaShop\Module\BlockWishList\ObjectModel;
```
Comment (Matt75) [inline src/ObjectModel/Statistics.php:3]: No namespace for ObjectModel, should always be in global namespace

Comment (zalexki) [reply src/ObjectModel/Statistics.php:3]: no need to review this is not finished and also missing a lot of clean of the php classes (i will ask a review when donne ;) )

Comment (Matt75) [reply src/ObjectModel/Statistics.php:3]: Add label WIP or set it as Draft ;)

Comment (zalexki) [reply src/ObjectModel/Statistics.php:3]: i cannot add label :D and this is said in description ;)

**Comment by NeOMakinG** [comment]
Thanks @zalexki , I'll start the integration then!

FYI : Eslint is fixed on my PR : https://github.com/PrestaShop/blockwishlist/pull/39

We'll need to rebase this one after the #39 merged

**Pull request closed without merging**